<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Recordatorios</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QFrame" name="frame">
      <property name="styleSheet">
       <string notr="true">background-image: url(imagenes/fondo.png);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="1">
        <widget class="QTabWidget" name="tabWidget">
         <property name="styleSheet">
          <string notr="true">background-image: url(imagenes/frame.png);
background-position: center;</string>
         </property>
         <widget class="QWidget" name="tab">
          <attribute name="title">
           <string>Tab 1</string>
          </attribute>
          <widget class="QPushButton" name="btnCorreo">
           <property name="geometry">
            <rect>
             <x>180</x>
             <y>320</y>
             <width>111</width>
             <height>24</height>
            </rect>
           </property>
           <property name="text">
            <string>Prueba Correo</string>
           </property>
          </widget>
          <widget class="QLabel" name="label">
           <property name="geometry">
            <rect>
             <x>40</x>
             <y>20</y>
             <width>49</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>Correo:</string>
           </property>
          </widget>
          <widget class="QLineEdit" name="correoInput">
           <property name="geometry">
            <rect>
             <x>40</x>
             <y>40</y>
             <width>391</width>
             <height>22</height>
            </rect>
           </property>
          </widget>
          <widget class="QLabel" name="label_2">
           <property name="geometry">
            <rect>
             <x>40</x>
             <y>80</y>
             <width>49</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>Mensaje:</string>
           </property>
          </widget>
          <widget class="QScrollArea" name="scrollArea">
           <property name="geometry">
            <rect>
             <x>40</x>
             <y>100</y>
             <width>391</width>
             <height>211</height>
            </rect>
           </property>
           <property name="widgetResizable">
            <bool>true</bool>
           </property>
           <widget class="QWidget" name="scrollAreaWidgetContents">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>389</width>
              <height>209</height>
             </rect>
            </property>
            <widget class="QTextEdit" name="mensajeInput">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>391</width>
               <height>211</height>
              </rect>
             </property>
            </widget>
           </widget>
          </widget>
         </widget>
         <widget class="QWidget" name="tab_2">
          <attribute name="title">
           <string>Tab 2</string>
          </attribute>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
